proxy/proxy/rhnAuthCacheClient.py

```diff
@@ -186,9 +191,9 @@ def __str__(self):
     s = Shelf(('localhost', 9999))
     s['1234'] = [1, 2, 3, 4, None, None]
     s['blah'] = 'testing 1 2 3'
-    print 'Cached object s["1234"] = %s' % str(s['1234'])
-    print 'Cached object s["blah"] = %s' % str(s['blah'])
-    print s.has_key("asdfrasdf")
+    print('Cached object s["1234"] = {}'.format(str(s['1234'])))
+    print('Cached object s["blah"] = {}'.format(str(s['blah'])))
+    print("asdfrasdf" in s)
```

Comment:
?

Reply:
That code is part of a old test, which seems is never executed during normal operation.